addons/web/controllers/main.py

```diff
@@ -1004,6 +1003,12 @@ def content_common(self, xmlid=None, model='ir.attachment', id=None, field='data
             response.set_cookie('fileToken', token)
         return response
 
+    @http.route(['/web/partner_image',
+        '/web/partner_image/<string:model>/<int:id>/<string:field>'], type='http', auth="public")
+    def content_image_partner(self, *args, **kwargs):
+        kwargs.pop('placeholder');
```

Comment:
- No semicolon
- `pop()` raises a KeyError when the key is missing, so it would have needed a default here
-  instead of popping `placeholder`, it seems simpler/safer to pass only the required parameters (model, id, field) to `content_image()`, and drop the kwargs, if we don't use them.
- seems like we could provide defaults for `model` and `field` and allow shorter URLs

@seb-odoo If you don't mind I'd like to keep the idea of extracting a common method for both controllers, plus entirely forbid passing a placeholder parameter on the routes. Let's continue reducing the number of supported query string parameters for `/web/image`.


@alexkuhn Don' t bother fixing it, I'll update the branch before merging, thanks for your work! :)